ambassador/kat/harness.py

```diff
@@ -183,7 +183,7 @@ def sanitize(obj):
 
 
 def abstract_test(cls: type):
-    cls.abstract_test = True
+    cls.abstract_test = True  # type: ignore
```

Comment:
I just now realized that all of these errors are happening because `kat` is now under `ambassador`. D'oh! 🙂 

In general, we should clean up the hinting that's resulting in the errors, rather than just telling `mypy` to ignore them. That could happen in a separate PR right after this one, if necessary.

Reply:
Yeah, I know, but I didn't have enough knowledge for fixing these warnings. I will try to do a proper fix in a subsequent PR...